backend/vault.go

```diff
@@ -154,14 +154,16 @@ func (c *client) renewalLoop() {
 		c.logger.Info("trying to login to vault again")
 		if err = c.vaultLogin(); err != nil {
 			vMetrics.updateVaultLoginErrorsTotalMetric()
+			c.logger.Info("login error, vault token not obtained")
```

Comment:
This should be an `Error` not `Info` and maybe also include the content of `err` to give more context?